IPython/core/tests/test_async_helpers.py

```diff
@@ -0,0 +1,37 @@
+"""
+Test for async helpers. 
+
+Should only trigger on python 3.5+ or will have syntax errors.
+"""
+
+import sys
+import nose.tools as nt
+from textwrap import dedent
+
+ip = get_ipython()
+iprs = lambda x: ip.run_cell(dedent(x))
+
+if sys.version_info > (3,5):
```

Comment:
Rather than making the test definitions themselves conditional, why not skip the tests?